src/sentry/spans/grouping/strategy/base.py

```diff
@@ -135,6 +135,32 @@ def loose_normalized_db_span_in_condition_strategy(span: Span) -> Optional[Seque
     return [cleaned]
 
 
+def join_regexes(regexes: Sequence[str]) -> str:
+    return r"(?:" + r")|(?:".join(regexes) + r")"
+
+
+DB_PARAMETRIZATION_PATTERN = re.compile(
+    join_regexes(
+        [
+            r"'(?:[^']|'')*?(?:\\'.*|'(?!'))",  # single-quoted strings
+            r"-?\b(?:[0-9]+\.)?[0-9]+(?:[eE][+-]?[0-9]+)?\b",  # numbers
+            r"\b(?:true|false)\b",  # booleans
+        ]
+    )
+)
+
+
+@span_op(["db", "db.query", "db.sql.query", "db.sql.active_record"])
+def parametrize_db_span_strategy(span: Span) -> Optional[Sequence[str]]:
+    """Something"""
```

Comment:
😂

Reply:
...missed that one 😂

Reply:
re: negative tests and unexpected literal tests - these are there on the `parametrize_db_span_strategy` unit tests:
```
("SELECT * FROM table123 WHERE id = %s", None),
("SELECT * FROM table_true WHERE deleted = %s", None),
```
and I used `LIMIT` instead of `OFFSET` but
```
("SELECT * FROM table LIMIT 1", ["SELECT * FROM table LIMIT %s"]),
```

Do those look sufficient or should I add more?

Good point about the loose IN tests, I'll add those to the `parametrize_db_span_strategy` tests as well.

BTW, re: stacking the loose IN strategy: I looked into it originally but I don't think it will work because, to my understanding, [we stop looking at strategies as soon as one returns a value](https://github.com/getsentry/sentry/blob/d6eb9fd46c5a7b1da23b710e30238bb5dc3f0075/src/sentry/spans/grouping/strategy/base.py#L71-L77). The end result is we can only really have one strategy per op.

Reply:
Gotcha! In that case LGTM, up to you on the test coverage! Resolving